source/unified-test-format/unified-test-format.rst

```diff
@@ -482,77 +480,21 @@ The structure of this object is as follows:
-    
-    For each entity name, the test runner MUST create the respective entity
-    with a type of "event list", as described in `Supported Entity Types`_.
-    If the entity already exists (such as from a previous
-    ``storeEventsAsEntities`` declaration from another client),
-    the test runner MUST raise an error.
-    
-    .. note: The implementation of ``storeEventsAsEntities`` is wholly
-      independent from ``observeEvents`` and ``ignoreCommandMonitoringEvents``.
-    
-    The test runner MUST set up an event subscriber for each event named.
-    The event subscriber MUST serialize the events it receives into a document,
-    using the documented properties of the event as field names, and append
-    the document to the list stored in the specified entity.
-    Additionally, the following fields MUST be stored with each
-    event document:
-    
-    - ``name``: The name of the event, such as ``PoolCreatedEvent``.
-      The name of the event MUST be the name used in the respective
-      specification that defines the event in question.
-    - ``observedAt``: The floating-point time since the Unix epoch
-      when the event was observed by the test runner.
-        
-    Currently, only the following
-    `CMAP <../connection-monitoring-and-pooling/connection-monitoring-and-pooling.rst>`__
-    and `command <../command-monitoring/command-monitoring.rst>`__
-    events MUST be supported:
-    
-    - PoolCreatedEvent
-    - PoolReadyEvent
-    - PoolClearedEvent
-    - PoolClosedEvent
-    - ConnectionCreatedEvent
-    - ConnectionReadyEvent
-    - ConnectionClosedEvent
-    - ConnectionCheckOutStartedEvent
-    - ConnectionCheckOutFailedEvent
-    - ConnectionCheckedOutEvent
-    - ConnectionCheckedInEvent
-    - CommandStartedEvent
-    - CommandSucceededEvent
-    - CommandFailedEvent
-    
-    The test runner MAY omit the ``command`` field for CommandStartedEvent
-    and ``reply`` field for CommandSucceededEvent.
-    
-    If an event field in the driver is of a type that does not directly map to
-    a BSON type, such as ``Exception`` for the ``failure`` field of
-    CommandFailedEvent, the test runner MUST convert values of that field
-    to one of the BSON types.
-    
-    If specification defining an event permits deviation in field names,
-    such as ``connectionId`` field for CommandStartedEvent, the test runner
-    SHOULD use the field names used in the specification when serializing
-    events to documents even if the respective field name is different in the
-    driver's event object.
-    
+  - ``storeEventsAsEntities``: Optional array of one or more
+    `storeEventsAsEntity`_ objects. Each object denotes an entity name and one
+    or more events to be collected and stored in that entity. See
+    `storeEventsAsEntity`_ for implementation details.
+
+    Note: the implementation of ``storeEventsAsEntities`` is wholly independent
+    from ``observeEvents`` and ``ignoreCommandMonitoringEvents``.
```

Comment:
@p-mongo: Note that `.. note:` is Sphinx syntax. I just realized that this wasn't being rendered by GitHub (and another RST previewing tool) when I tried to use it elsewhere for `serverApi.version`.